packages/babel-plugin/src/__tests__/__snapshots__/errors.test.tsx.snap

```diff
@@ -28,7 +28,7 @@ exports[`error handling should throw when referencing an identifier that isnt su
 `;
 
 exports[`error handling should throw when spreading an identifier that does not exist 1`] = `
-"unknown: Variable could not be found (5:21).
+"unknown: Variable could not be found or invalid attempt to spread an object (5:21).
```

Comment:
can we separate the error into two?

Reply:
🤔 I knew it. 😛 You will point this out. Let me see what I can do. It's difficult as I have removed lot of code around binding and reused things from `getInterpolation`.